std/math.d

```diff
@@ -6832,6 +6832,149 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     assert(approxEqual(pow(twoI, three), 8.0));
 }
 
+/** Computes the value of a positive integer x, raised to the power n, modulo m.
+ *
+ * All values have to be unsigned types, otherwise it will raise a compilation error.
+ *
+ */
+Unqual!(Largest!(F, H)) powmod(F, G, H)(F x, G n, H m) @nogc @safe pure nothrow
```

Comment:
Template functions should not rely on attributes inference and have unittest that verifies that the correct set of attributes is inferred -> drop `@nogc @safe pure nothrow` here and add the missing attributes to the unittest below.